feature/staticroute/TwoPortConfig/static_route_test.go

```diff
@@ -0,0 +1,101 @@
+package gnmi_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestGNMI(t *testing.T) {
+
+	// Configure a DUT
+
+	dut := ondatra.DUT(t, "dut")
+	dp1 := dut.Port(t, "port1")
+	dp2 := dut.Port(t, "port2")
+
+	ifCfg1 := &telemetry.Interface{
+		Name:        ygot.String(dp1.Name()),
+		Description: ygot.String("From Ixia"),
+	}
+
+	ifCfg1.GetOrCreateSubinterface(0).
+		GetOrCreateIpv4().
+		GetOrCreateAddress("192.0.2.10").
+		PrefixLength = ygot.Uint8(31)
+
+	ifCfg2 := &telemetry.Interface{
+		Name:        ygot.String(dp2.Name()),
+		Description: ygot.String("To Ixia"),
+	}
+
+	ifCfg2.GetOrCreateSubinterface(0).
+		GetOrCreateIpv4().
+		GetOrCreateAddress("192.0.2.20").
+		PrefixLength = ygot.Uint8(31)
+
+	dut.Config().Interface(dp1.Name()).Update(t, ifCfg1)
+	dut.Config().Interface(dp2.Name()).Update(t, ifCfg2)
+
+	Ni1 := &telemetry.NetworkInstance{}
+
+	Ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").
+		GetOrCreateStatic("10.0.0.0/24").
+		GetOrCreateNextHop("h1").NextHop = telemetry.UnionString("192.0.2.10")
+
+	dut.Config().NetworkInstance("default").Update(t, Ni1)
+
+	//  Configure an ATE
+
+	ate := ondatra.ATE(t, "ate")
+	ap1 := ate.Port(t, "port1")
+	ap2 := ate.Port(t, "port2")
+
+	top := ate.OTG().NewConfig(t)
+
+	top.Ports().Add().SetName(ap1.ID())
```

Comment:
Again, we might want to refactor this in loop that does both ports. And like above, we could look at [`Attributes`](https://github.com/openconfig/featureprofiles/blob/71497a3b4c5d5806e8425f5b7ee748231d5a53ec/internal/attrs/attrs.go#L35).